src/vs/workbench/contrib/multiDiffEditor/browser/multiDiffEditorInput.ts

```diff
@@ -98,10 +98,10 @@ export class MultiDiffEditorInput extends EditorInput implements ILanguageSuppor
 			const resources = this.resources.read(reader);
 			const label = this.label ?? localize('name', "Multi Diff Editor");
 			if (resources) {
-				this._name = label + localize({
-					key: 'files',
-					comment: ['the number of files being shown']
-				}, " ({0} files)", resources.length);
+				const fileLabel = resources.length === 1 ?
```

Comment:
I would actually argue that we should be pulling in the `label` into the `fileLabel` for better translation support: `"{0} ({1} files)"`